judge/template_context.py

```diff
@@ -30,6 +30,7 @@ def get_resource(request):
         'FONTAWESOME_CSS': settings.FONTAWESOME_CSS,
         'DMOJ_SCHEME': scheme,
         'DMOJ_CANONICAL': settings.DMOJ_CANONICAL,
+        'SELECT2_THEME': settings.DMOJ_SELECT2_THEME,
```

Comment:
For consistency with the other settings, this should be

```suggestion
        'DMOJ_SELECT2_THEME': settings.DMOJ_SELECT2_THEME,
```

Reply:
Done